service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/deser/PathParamSerializers.java

```diff
@@ -62,33 +62,39 @@ public Param deserialize(PSequence<String> parameters) {
     }
 
     /**
-     * A String id serializer.
+     * A String path param serializer.
      */
     public static final PathParamSerializer<String> STRING = required("String", Function.identity(),
             Function.identity());
 
     /**
-     * A Long id serializer.
+     * A Long path param serializer.
      */
     public static final PathParamSerializer<Long> LONG = required("Long", Long::parseLong, l -> l.toString());
 
     /**
-     * An Integer id serializer.
+     * An Integer path param serializer.
      */
     public static final PathParamSerializer<Integer> INTEGER = required("Integer", Integer::parseInt, i -> i.toString());
 
     /**
-     * A Boolean id serializer.
+     * A Boolean path param serializer.
      */
     public static final PathParamSerializer<Boolean> BOOLEAN = required("Boolean", Boolean::parseBoolean,
             b -> b.toString());
 
     /**
+     * A UUID path param serializer.
+     */
+    public static final PathParamSerializer<UUID> UUID = required("UUID", java.util.UUID::fromString,
+            u -> u.toString());
```

Comment:
Great! 👍 I had just been thinking we should provide this out of the box.

Would also be nice to add a convenience for creating Enum path param serializers.